runtime/runtime_impl_jetty9/src/main/java/com/google/apphosting/runtime/jetty9/JettyServletEngineAdapter.java

```diff
@@ -141,10 +140,10 @@ public void start(String serverInfo, ServletEngineAdapter.Config runtimeOptions)
         if (Boolean.getBoolean(HTTP_CONNECTOR_MODE)) {
           logger.atInfo().log("Using HTTP_CONNECTOR_MODE to bypass RPC");
           appVersionKey = AppVersionKey.fromAppInfo(appinfo);
-          JettyHttpProxy.insertHandlers(server);
           AppVersion appVersion = appVersionHandlerMap.getAppVersion(appVersionKey);
           server.insertHandler(
               new JettyHttpHandler(runtimeOptions, appVersion, appVersionKey, appInfoFactory));
+          JettyHttpProxy.insertHandlers(server);
```

Comment:
Do you think we should change the readability of the code for both this and Jetty12 i.e. it _says_ that it is `insertHandler` when really it is _setting_ a new handler (after doing handler chain). Ideally, I would personally prefer to use only  `insertHandler` every as it improves readability and also becaues it calls `setHandler` internally either ways becuase right now the order of the chain requires 2-3 reads to understand. Let me know your thoughts on this.

Reply:
I was doing a bunch of cleanups around this and it got a bit out of scope for this PR, so I am planning to do a dedicated PR to clean this up and leave this one focused on the fix.

Reply:
see https://github.com/GoogleCloudPlatform/appengine-java-standard/pull/309